introduction.md

```diff
@@ -1 +1,6 @@
 I am jeffweiss. My favorite Code School path is Electives.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?